### ISSUE TITLE
Change from double to single click for opening the details view of an entry

### ISSUE THREAD

**Issue by claell** [JabRef/jabref#8772]
**Is your suggestion for improvement related to a problem? Please describe.**
From Citavi, I am used to only need a single click to open the details view of an entry. In JabRef, one needs a double click.

**Describe the solution you'd like**
Possibly think about changing to single click for that action.

**Additional context**
Not sure about that myself. Maybe there are good reasons for the current behavior.

**Comment by ThiloteE** [comment]
Yes, there was a reason and discussion, but I can't find the corresponding issue.

**Comment by mlep** [comment]
If, from the entry table, a single left click opens the entry: 
- How to select an entry and then perform an action from the menu (such as cleanup entries)?
- How to select multiple entries?

**Comment by claell** [comment]
Alright.

@ThiloteE I searched for that issue, but also couldn't find one.

Regarding the two aspects @mlep:
1. An entry could be selected (with as result opening the entry editor) and one can still perform an action from the menu
2. `Ctrl + Click`, just how it currently works already

**Comment by u7306160** [comment]
Hi, my group plan to work on this issue and do it by the end of October 2022. This is our first issue on an open-source project. Could you assign this issue to me, please?

**Comment by Siedlerchr** [comment]
@u7306160  This issue is **not** ready to take because it needs prior discussion and a decision if we really want to do this. I currently don't think so.
